codalab/rest/bundles.py

```diff
@@ -426,6 +428,48 @@ def _fetch_locations():
     return dict(data=uuids_to_locations)
 
 
+@get('/bundles/<bundle_uuid:re:%s>/locations/', apply=AuthenticatedProtectedPlugin())
+def _fetch_bundle_locations(bundle_uuid: str):
+    """
+    Returns a list of BundleLocations associated with the given bundle
+
+    Query parameters:
+    - `bundle_uuid`: Bundle UUID to get the locations for
+    """
+    bundle_locations = local.model.get_bundle_locations(bundle_uuid)
+    return BundleLocationListSchema(many=True).dump(bundle_locations).data
+
+
+@post('/bundles/<bundle_uuid:re:%s>/locations/', apply=AuthenticatedProtectedPlugin())
+def _add_bundle_location(bundle_uuid: str):
+    """
+    Adds a new BundleLocation to a bundle
+
+    Query parameters:
+    - `bundle_uuid`: Bundle UUID corresponding to the new location
+    - `bundle_store_uuid`: Bundle Store UUID corresponding to the new location
```

Comment:
This is not a query parameter, only bundle_uuid is. You should instead specify what the POST body should be equal to (the data structue of it).